dces-drc-integration/src/test/java/uk/gov/justice/laa/crime/dces/integration/service/ContributionServiceTest.java

```diff
@@ -58,11 +57,11 @@
 import static org.springframework.http.HttpStatus.OK;
 
 @ExtendWith(SoftAssertionsExtension.class)
-@SpringBootTest
-@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+//@SpringBootTest
+//@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
Probably missed, but should remove these commented lines.

Reply:
Removed the commented lines.